lib/tasks/app_setup.rake

```diff
@@ -0,0 +1,20 @@
+namespace :app do
```

Comment:
FYI, `app:` is used by Rails. As of Rails 6: `app:template` and `app:update` are defined. This won't conflict with `app:setup` but we might need an update script in the future.

Reply:
oh snap, thanks for the heads up! I will change it so we don't have issues.

Reply:
in the future `app_initializer` could be a project name but this keeps it general for now